index.d.ts

```diff
@@ -0,0 +1,40 @@
+import { 
+    Event, 
+    EventTarget, 
+    AddEventListenerOptions, 
+    EventListenerOptions,
+    EventTargetListener
+ } from "event-target-shim";
+
+export as namespace AbortControllerShim;
+
+export type AbortSignalListener = (this: AbortSignal, ev: Event) => any;
```

Comment:
```suggestion
export type AbortSignalListener = (this: AbortSignal, ev: any) => any;
```

Would you make the `ev` type to `any`?
It will make this PR being able to merge soonly.